tests/integrations/raftstore/test_flashback.rs

```diff
@@ -0,0 +1,203 @@
+// Copyright 2022 TiKV Project Authors. Licensed under Apache-2.0.
+
+use std::time::Duration;
+
+use engine_traits::CF_DEFAULT;
+use test_raftstore::*;
+use txn_types::WriteBatchFlags;
+
+#[test]
+fn test_flahsback_for_parameters() {
```

Comment:
I think this test is unnecessary since the schedule, read and write blocking tests below can cover the case.